### PR TITLE
Add an interpreter target to python toolchain

### PR DESCRIPTION
**Pull request by benradf**
My attempt at adding an `interpreter` target for the python toolchain as requested in #255. I think this maintains the separation between the toolchain workspace and the nix workspace you mentioned @aherrmann, but please let me know if not.

### CODE REVIEW

File: toolchains/python/python.bzl [hunk 1]
```diff
@@ -36,20 +44,25 @@ toolchain(
...
+        python3_runtime = label_string(python3_runtime),
         exec_constraints = exec_constraints,
         target_constraints = target_constraints,
     ))
 
+    repository_ctx.file("defs.bzl", executable = False, content = """
+# Generated by rules_nixpkgs/toolchains/python/python.bzl
+interpreter = "{}"
+    """.format(repository_ctx.attr.python3_repo + ":bin/python"))
+
```
Comment (aherrmann) [inline toolchains/python/python.bzl:57]: The case where `repository_ctx.attr.python3_repo` is empty doesn't seem to be handled here. In that case it should probably fall back to the python2 one.

**Review by aherrmann — CHANGES_REQUESTED** [review]
Looks good, thank you!

One comment.

File: toolchains/python/python.bzl (hunk 1 above)
Comment (benradf) [reply toolchains/python/python.bzl:57]: Yeah, good point. Will do this now.

**Review by aherrmann — APPROVED** [review]
Thank you, looks good!

Closes #255

**Pull request merged**